float32 stick1_h
float32 stick1_v
float32 stick2_h
float32 stick2_v
int32 intStick_h
int32 intStick_v
uint8 square
uint8 cross
uint8 circle
uint8 triangle
uint8 L1
uint8 R1
uint8 L2
uint8 R2
uint8 SE
uint8 ST
uint8 stick1
uint8 stick2
uint32 mode_type1
uint32 mode_type2
uint8 on_state
